components/homme/src/theta-l_kokkos/cxx/CaarFunctorImpl.hpp

```diff
@@ -362,10 +362,7 @@ struct CaarFunctorImpl {
       GPTLstop("caar compute");
     }
 
-    GPTLstart("caar dp3d");
-    Kokkos::parallel_for("caar loop dp3d limiter", m_policy_dp3d_lim, *this);
-    Kokkos::fence();
```

Comment:
i removed this fence without thinking. new limiter does not have a fence, is fence needed after the limiter?

Reply:
A fence is needed if there is a data dependency in the adjacent kernels.

Reply:
since it is only about loops in vertical and there is a team barrier, i assume it is not needed here.

Reply:
Here you don't need a fence because there's one above, line 361 or 355.